package.json

```diff
@@ -10,16 +10,18 @@
   "license": "MIT",
   "repository": "jsdom/whatwg-url",
   "dependencies": {
+    "stable": "^0.1.6",
     "tr46": "~0.0.3",
     "webidl-conversions": "^3.0.0"
   },
   "devDependencies": {
     "eslint": "^2.6.0",
     "istanbul": "~0.4.3",
+    "jsdom": "^10.1.0",
     "mocha": "^2.2.4",
+    "node-fetch": "^2.0.0-alpha",
     "recast": "~0.10.29",
-    "request": "^2.55.0",
```

Comment:
I'd kind of prefer sticking with what's been working so far. It's only at development time.